data-protocols/ids/ids-api-multipart-dispatcher-v1/src/main/java/org/eclipse/dataspaceconnector/ids/api/multipart/dispatcher/sender/MultipartCatalogDescriptionRequestSender.java

```diff
@@ -55,10 +55,41 @@ public MultipartCatalogDescriptionRequestSender(@NotNull String connectorId,
                                                     @NotNull ObjectMapper objectMapper,
                                                     @NotNull Monitor monitor,
                                                     @NotNull IdentityService identityService,
-                                                    @NotNull TransformerRegistry transformerRegistry) {
+                                                    @NotNull IdsTransformerRegistry transformerRegistry) {
         super(connectorId, httpClient, objectMapper, monitor, identityService, transformerRegistry);
     }
 
+    private static void createOfferResourcesFromProperties(ResourceCatalog catalog, ObjectMapper mapper) {
```

Comment:
I would keep private methods on the bottom of the class (and keep them non-static if not needed)